src/meta/client/MetaClient.cpp

```diff
@@ -26,12 +25,12 @@ MetaClient::MetaClient(std::shared_ptr<folly::IOThreadPoolExecutor> ioThreadPool
                        std::vector<HostAddr> addrs,
                        HostAddr localHost,
                        ClusterID clusterId,
-                       bool sendHeartBeat)
+                       bool inStoraged)
     : ioThreadPool_(ioThreadPool)
     , addrs_(std::move(addrs))
     , localHost_(localHost)
     , clusterId_(clusterId)
-    , sendHeartBeat_(sendHeartBeat) {
+    , inStoraged_(inStoraged) {
```

Comment:
So how to update the cache on query engine side.

Reply:
Now, sendHeartBeat is enabled by default, either on the query engine or the storage engine side. Whether to update the local cache is determined after each heartbeat based on the last update time of the metad. Only hosts where inStoraged is true will be recorded on the meta server.